### PR TITLE
Feature/us5447 update attribute structures

### PR DESCRIPTION
**Pull request by ilpadre**

### CODE REVIEW

File: Gateway/MinistryPlatform.Translation/Repositories/Interfaces/IAttributeRepository.cs [hunk 1]
```diff
@@ -6,7 +6,6 @@ namespace MinistryPlatform.Translation.Repositories.Interfaces
     public interface IAttributeRepository
     {
         List<MpAttribute> GetAttributes(int? attributeTypeId);
-        List<MpAttribute> GetAttributesByFilter(string filter);
```
Comment (Doug-Shannon) [inline Gateway/MinistryPlatform.Translation/Repositories/Interfaces/IAttributeRepository.cs]: Did you mean to remove this?

File: Gateway/crds-angular/Services/ObjectAttributeService.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
```
Comment (Doug-Shannon) [inline Gateway/crds-angular/Services/ObjectAttributeService.cs]: Did you mean to add this?  If so what is it for?

File: Gateway/crds-angular/Services/ObjectAttributeService.cs [hunk 3]
```diff
@@ -88,6 +89,11 @@ public ObjectAllAttributesDTO GetObjectAttributes(string token, int objectId, Mp
                 }
 
                 var objectAttributeType = attributeTypesDictionary[mpObjectAttribute.AttributeTypeId];
+
+                if (!objectAttributeType.Attributes.Exists(x => x.AttributeId == mpObjectAttribute.AttributeId))
```
Comment (Doug-Shannon) [inline Gateway/crds-angular/Services/ObjectAttributeService.cs]: This is dense, maybe add a comment to clarify

File: Gateway/MinistryPlatform.Translation/Repositories/Interfaces/IAttributeRepository.cs (hunk 1 above)
Comment (JoeKer1) [inline Gateway/MinistryPlatform.Translation/Repositories/Interfaces/IAttributeRepository.cs]: yup

**Pull request merged**